auth/lib/utils/totp_util.dart

```diff
@@ -4,7 +4,7 @@ import 'package:otp/otp.dart' as otp;
 import 'package:steam_totp/steam_totp.dart';
 
 String getOTP(Code code) {
-  if (code.issuer.toLowerCase() == 'steam') {
```

Comment:
Other than what @ua741 suggested, you will also have to modify the logic of _getType and _getDigits in code.dart to support both the issuers.

Reply:
Ok, let's proceed with that!